### PR TITLE
ar71xx: add support for gl-mifi

### PR DESCRIPTION
**Pull request by zhao-gang**
This add initial support for gl-mifi.
Product page: http://www.gl-inet.com/mifi/

### CODE REVIEW

File: target/linux/ar71xx/base-files/etc/board.d/01_leds [hunk 1]
```diff
@@ -275,6 +275,13 @@ gl-inet)
 	ucidef_set_led_wlan "wlan" "WLAN" "gl-connect:red:wlan" "phy0tpt"
 	;;
 
+gl-mifi)
```
Comment (blogic) [inline target/linux/ar71xx/base-files/etc/board.d/01_leds]: this is not alphabetically ordered..

**Comment by blogic** [comment]
merged, thanks !

**Comment by kristrev** [comment]
I just tested this commit and noticed an error, causing the boot to fail. There is a ";;" missing in ar71xx.sh.

**Comment by blogic** [comment]
pushed a fix just now, please retest

**Comment by kristrev** [comment]
Looks good, thanks.

**Pull request closed without merging**